src/pages/Auth/Auth.css

```diff
@@ -0,0 +1,29 @@
+
+ .bg-img{
+    width: 50%; 
+    height:  90%; 
+    object-fit: cover;
+}
+
+.input{
+    width: 22rem;
+}
+
+.divider{
+    color: #A14F57;
+    font-weight: 800;
+}
+
+.auth-form{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap:10px;
+}
+ 
+@media screen and (max-width: 600px) {
```

Comment:
Suggestion: you can use ems for media query

Reply:
ok